node/src/types/sync_leap.rs

```diff
@@ -148,6 +168,34 @@ impl SyncLeap {
             )
     }
 
+    pub(crate) fn global_states_for_sync_across_upgrade(&self) -> Option<GlobalStatesMetadata> {
```

Comment:
We should soon be able to merge [a PR](https://github.com/casper-network/casper-node/pull/3726) that adds a full battery of unit tests to `SyncLeap`.
Do you mind adding UTs for the new functions?

Reply:
Addressed in 03d9f13